docker/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -x
+#########Change ME###############
+export PYVER=2.7
+export PYV=27
+export CUDA_VERSION=10
+export NVIDIA_BUILD_ID=12345
+export DEPS_IMAGE=dali_cu${CUDA_VERSION}.deps
+export BUILDER=dali_${PYV}_cu${CUDA_VERSION}.build
+export RUN_IMG=dali_${PYV}_cu${CUDA_VERSION}.run
+#################################
+pushd ../
+docker build -t ${DEPS_IMAGE} --build-arg "USE_CUDA_VERSION=${CUDA_VERSION}" -f Dockerfile.deps .
+echo "Build image:" ${BUILDER}
+test $? -eq 0 && docker build -t ${BUILDER} --build-arg "DEPS_IMAGE_NAME=${DEPS_IMAGE}" --build-arg "PYVER=${PYVER}" --build-arg "PYV=${PYV}" --build-arg "NVIDIA_BUILD_ID=${NVIDIA_BUILD_ID}" .
+echo "Runner image:" ${RUN_IMG}
+echo "You can run this image with DALI installed inside, keep in mind to install neccessary FW package as well"
+if [ ${CUDA_VERSION} == "9" ] ; then
+    export CUDA_IMAGE_NAME="nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04"
+elif [ ${CUDA_VERSION} == "10" ] ; then
+    export CUDA_IMAGE_NAME="nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04"
+else
+    echo "**************************************************************"
+    echo "Not supported CUDA version"
+    echo "**************************************************************"
+fi
+echo ${CUDA_VERSION}
+echo ${CUDA_IMAGE_NAME}
+
+test $? -eq 0 && docker build -t ${RUN_IMG} --build-arg "BUILD_IMAGE_NAME=${BUILDER}" --build-arg "CUDA_IMAGE_NAME=${CUDA_IMAGE_NAME}" --build-arg "PYVER=${PYVER}" --build-arg "PYV=${PYV}" -f Docker_run_cuda .
```

Comment:
Don't we want to use nvidia-docker for running with cuda?

Reply:
`nvidia-docker` is basically only for running. For building it doesn't matter